store/airbus_callback.go

```diff
@@ -0,0 +1,59 @@
+package store
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/myntra/goscheduler/constants"
+)
+
+func (h *AirbusCallback) GetType() string {
+	return constants.AirbusCallback
+}
+
+func (h *AirbusCallback) GetDetails() (string, error) {
+	details, err := json.Marshal(h)
+	return string(details), err
+}
+
+func (h *AirbusCallback) Marshal(m map[string]interface{}) error {
+	eventName, ok := m["call_back_url"].(string)
```

Comment:
Are we using same key `call_back_url` even in to store eventName? Since event name is not a URL, shouldn't it better to use some other key or some common key name which is independent of event type.

Reply:
This is done to provide the backward compatibility support in MySS. This will be removed in subsequent releases. The GoScheduler Schema uses generic names like [callback_type](https://github.com/myntra/goscheduler/blob/main/cassandra/cassandra.cql#L8), [callback_details](https://github.com/myntra/goscheduler/blob/main/cassandra/cassandra.cql#L9)